packages/xstate-graph/src/types.ts

```diff
@@ -171,7 +197,14 @@ export interface TraversalOptions<TState, TEvent extends EventObject>
    *
    * @default `Infinity`
    */
-  traversalLimit?: number;
+  traversalLimit: number;
+  fromState: TState | undefined;
+  /**
+   * When true, traversal of the adjacency list will stop
```

Comment:
The concept of adjacencies could be explained. I'm not sure what is the difference between an adjacency map and an adjacency list. Are those relevant to the public API/to the user? Or are those implementation details? By mentioning them in the JS Doc we will surface them to the users as this will appear in the IDE tooltips etc

Reply:
Tried to clarify it.

An adjacency list is the state -> event -> nextState relations. Example, given this machine:

```js
const machine = createMachine({
  initial: 'green',
  states: {
    green: { on: { TIMER: 'yellow' } },
    yellow: { on: { TIMER: 'red' } },
    red: { on: { TIMER: 'green' } }
  }
});
```

An adjacency list (array) would look like this (with states, not state values):
```
[
  { state: 'green', event: 'TIMER', nextState: 'yellow' }, // green → yellow
  { state: 'yellow', event: 'TIMER', nextState: 'red' }, // yellow → red
  { state: 'red', event: 'TIMER', nextState: 'green' } // red → green
]
```